app/helpers/carts_helper.rb

```diff
@@ -11,4 +11,12 @@ def init_cart
   def load_products_into_cart
     @products = Product.by_ids @cart.keys
   end
+
+  def total_price_into_cart
+    @sum = 0
+    @products.each do |product|
+      total = product.price * @cart[product.id.to_s]
+      @sum += total
```

Comment:
```suggestion
      @sum +=  product.price * @cart[product.id.to_s]
```
cơ mà sao ko dùng lun method sum hoặc inject cụ

Reply:
Dạ em ready ạ.